mesop/examples/web_component/google_maps/google_maps_app.py

```diff
@@ -0,0 +1,66 @@
+"""Proof of concept of a webcomponent wrapper that adds Google Maps support.
+Note: Does not implement full interactions with a map. This is meant as a
+    proof of concept that others can build upon for additional interactions with
+    a web map.
+"""
+
+from pydantic import BaseModel
+from typing import Tuple
+
+import mesop as me
+import mesop.labs as mel
+
+from mesop.examples.web_component.google_maps import(
```

Comment:
`mesop.examples.web_component.google_maps.google_maps_component` -> Also just made this adjustment for you since I was making the edits.